dids/module.md

```diff
@@ -16,6 +20,58 @@ val keyManager = InMemoryKeyManager()
 val did = DidKey.create(keyManager)
 ```
 
+**Resolve**
+
+```kt
```

Comment:
I have ```kotlin in the other module.md, should be the same?

Reply:
this file was already using kt, but don't matter to me